qiskit/chemistry/core/hamiltonian.py

```diff
@@ -47,7 +47,7 @@ class QubitMappingType(Enum):
     warnings.warn('The chemistry.core.QubitMappingType class is deprecated as of Qiskit Aqua '
```

Comment:
I am not sure we need these deprecations in here. The whole Hamiltonian is deprecated and hence these that were used with it. Since Hamiltonian will show a deprecation warning I think that suffices. Otherwise it was noted that we now have a bunch of deprecation warnings in the test just via the import of chemistry.core
@manoelmarques Maybe you have some comment since this was your observation around the build outcome

Reply:
Those enums are used as default values in Hamiltonian so as soon as hamiltonian.py gets loaded, it print the enum deprecation msgs. The hamiltonian.py is loaded from the chemistry.core init so just importing chemistry.core will print the enum deprecation.

Reply:
I see the QubitMappingType in the FermionicTransformation in the Excited states PR has has a name change so even more I think we should just remove these deprecation messages from the enums leaving it just on the Hamiltonian where you would use these.